tests/components/owntracks/test_device_tracker.py

```diff
@@ -37,7 +37,7 @@
 
 TEST_ZONE_LAT = 45.0
 TEST_ZONE_LON = 90.0
-TEST_ZONE_DEG_PER_M = 0.0000127
+TEST_ZONE_DEG_PER_M = 0.0000010
```

Comment:
This change is probably not good, changed to make tests using the `ZONE_EDGE` constant pass.